Flask/tests/test_routes.py

```diff
@@ -79,4 +79,76 @@ def test_registration(self):
         result = self.routeActions.registerAction(email, password, role)
 
         self.assert_template_used("register.html")
-        self.assert_context("registrationCheck", self.dbHelper.REGISTRATION_SUCCESS)
\ No newline at end of file
+        self.assert_context("registrationCheck", self.dbHelper.REGISTRATION_SUCCESS)
+
+    def test_registration_success_on_valid_creator_email(self):
```

Comment:
This test seems like a duplicate of the test above.

I'd remove the test above, since this one is better named.